private/pdk-project.ts

```diff
@@ -182,86 +183,50 @@ export abstract class PDKProject extends JsiiProject {
       ],
     });
 
+    this.nx = NxProject.ensure(this);
+    options.nx && this.nx.merge(options.nx);
+
+    // Make sure this is after NxProject so targets can be updated after inference
     this.pdkRelease = new PDKRelease(this, options.publishConfig);
     new PDKDocgen(this);
-
-    if (options.nx) {
-      this.nx = options.nx;
-    }
-  }
-
-  /**
-   * Builds a command to execute in a particular workspace.
-   *
-   * @param args args to append to command.
-   * @protected
-   */
-  protected buildExecuteInWorkspaceCommand(...args: string[]) {
-    switch (this.package.packageManager) {
-      case NodePackageManager.YARN:
-      case NodePackageManager.YARN2:
-        return `yarn workspace ${args.join(" ")}`;
-      case NodePackageManager.PNPM:
-        return `pnpm --filter ${args.join(" ")}`;
-      default:
-        return `npx -p ${args.join(" ")}`;
-    }
-  }
-
-  /**
-   * Builds a command to execute using current package manager (npx, yarn, pnpm).
-   *
-   * @param args args to append to command.
-   * @protected
-   */
-  protected buildExecuteCommand(...args: string[]) {
-    switch (this.package.packageManager) {
-      case NodePackageManager.YARN:
-      case NodePackageManager.YARN2:
-        return `yarn run ${args.join(" ")}`;
-      case NodePackageManager.PNPM:
-        return `pnpm exec ${args.join(" ")}`;
-      default:
-        return `npx ${args.join(" ")}`;
-    }
   }
 
-  /**
-   * Get Nx project configuration.
-   *
-   * If project does not have explicit Nx configuration, the workspace defaults
-   * will be returned.
-   *
-   * @see https://nx.dev/reference/project-configuration
-   */
-  public get nx(): Nx.ProjectConfig | undefined {
-    return this.manifest.nx || cloneDeep({ targets: NX_TARGET_DEFAULTS });
-  }
-
-  /**
-   * Set Nx project configuration.
-   *
-   * This will overwrite the entire configuration and replace any workspace
-   * defaults of same key.
-   *
-   * @see https://nx.dev/reference/project-configuration
-   */
-  public set nx(config: Nx.ProjectConfig | undefined) {
-    this.package.addField("nx", config);
-  }
-
-  /**
-   * Override specific Nx config value for specific path.
-   * @param path Key path to override value
-   * @param value Value to override
-   * @param {boolean} [append=false] Indicates if array values are appended to, rather than overwritten.
-   */
-  public nxOverride(path: string, value: any, append?: boolean): void {
-    const nx = cloneDeep(this.nx);
-    overrideField(nx, path, value, append);
-
-    this.nx = nx;
-  }
+  // /**
```

Comment:
Any reason this just can't be deleted?

Reply:
Oh it should have been, not sure how missed that